newIDE/app/src/UI/Theme/NordTheme/theme.json

```diff
@@ -49,6 +49,44 @@
         }
       }
     },
+    "search": {
+      "default": {
+        "background-color": {
+          "value": "#494952",
+          "comment": "Palette/Grey/70"
+        },
+        "text-color": {
+          "value": "#C5C5C9",
+          "comment": "Palette/Grey/30"
```

Comment:
should those comments be removed for non official themes?

Reply:
we could, and yet there're using the official theme colors, so I thought it's not a bad idea to keep them (in case someone in the future want to change them)